InventoryBuilder.py

```diff
@@ -231,7 +244,7 @@ def get_token(self, target):
                                      headers=headers)
             try:
                 return response.json()["token"]
-            except AttributeError as ar:
-                print("There is no attribute token!", ar.args)
-        except (HTTPError, KeyError) as err:
-            print("Request failed: ", err.args)
+            except AttributeError:
+                raise AttributeError("There is no attribute token!")
```

Comment:
please check for return code. I was currently running into this error, which has completely being masked:
`{'message': 'Password Policy Violated. Reason: "Password expired: 0 day(s) ago. Password needs to be changed to proceed".', 'httpStatusCode': 400, 'apiErrorCode': 1513}`

In case of a none 200 return code I'd expect to see the whole information above.

Current code produces this: 
`TypeError: can only concatenate str (not "NoneType") to str` because there is no `token` in the response and therefore it is `None`.